Maman11/Q2/my_suffix.c

```diff
@@ -0,0 +1,45 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <stdbool.h>
+#include <string.h>
+#define STRING_LENGTH 127
+
+int suffix(char str[], char c){
+    bool read = false;
+    int suffix = 0, len = strlen(str), i = 0;
+    while (str[i]){
+        if (c == str[i]){
```

Comment:
Again if you will use a for loop and a cleaner way all the functions would be cleaner and better